src/realm/cluster.cpp

```diff
@@ -675,9 +675,12 @@ ref_type Cluster::insert(ObjKey k, const FieldValues& init_values, ClusterNode::
     }
 
     return ret;
+error:
+    throw KeyAlreadyUsed(
```

Comment:
my preference would be to use a lambda or helper function instead of jumping (not blocking)

Reply:
I'll do that. I am still living in the past, it seems.